hardhat-kadena-plugin/packages/solidity-example/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "hardhat-project",
+    "devDependencies": {
+        "@nomicfoundation/hardhat-chai-matchers": "^2.0.0",
+        "@nomicfoundation/hardhat-ethers": "^3.0.0",
+        "@nomicfoundation/hardhat-ignition": "^0.15.9",
+        "@nomicfoundation/hardhat-ignition-ethers": "^0.15.0",
+        "@nomicfoundation/hardhat-network-helpers": "^1.0.0",
+        "@nomicfoundation/hardhat-toolbox": "^5.0.0",
+        "@nomicfoundation/hardhat-verify": "^2.0.12",
+        "@nomicfoundation/ignition-core": "^0.15.9",
+        "@openzeppelin/contracts": "^5.2.0",
+        "@typechain/ethers-v6": "^0.5.0",
```

Comment:
References to typechain, types, typescript, etc can be removed since typescript is not being used in this project. This was also removed from the demo solidity dir in the main branch.